packages/webworker/lumberjack/src/lib/formatting/lumberjack-format-log.ts

```diff
@@ -3,6 +3,24 @@ import { LumberjackLog } from '../logs/lumberjack.log';
 
 import { utcTimestampFor } from './utc-timestamp-for';
 
+/**
+ * Default function used by Lumberjack to format the Lumberjack logs.
+ *
+ * This function formats a LumberjackLog object into a string with specific format:
+ * "<level> <timestamp> [<scope>] <message>". If the scope is not provided, it will be omitted.
```

Comment:
suggestion: Rephrase
```suggestion
 * Format a {@link LumberjackLog} into a string of the format:
 * `"<level> <timestamp> [<scope>] <message>"`
 * If a scope is not provided, it will be omitted.
 *
 * @remarks Default function used by Lumberjack to format Lumberjack log.
```